lindera-cli/Cargo.toml

```
@@ -12,13 +12,13 @@ categories = ["text-processing"]
 license = "MIT"
 
 [features]
-default = ["ipadic"]  # ipadic is the default feature
-full = ["ipadic", "unidic", "ko-dic", "cc-cedict"]
-cjk = ["ipadic", "ko-dic", "cc-cedict"]
-ipadic = ["lindera/ipadic"]  # Japanese dictionary
-unidic = ["lindera/unidic"]  # Japanese dictionary
-ko-dic = ["lindera/ko-dic"]  # Korean dictionary
-cc-cedict = ["lindera/cc-cedict"]  # Chinese dictionary
+default = []  # No directories included
+full = ["ipadic", "unidic", "ko-dic", "cc-cedict"]  # All dictionaries included
+cjk = ["ipadic", "ko-dic", "cc-cedict"] # CJK dictionaries included
```

CJK の並び順の方がよかったりするのかな（些末 of 些末
```suggestion
cjk = ["cc-cedict", "ipadic", "ko-dic"] # CJK dictionaries included
```